python/ray/train/v2/tests/test_v2_api.py

```diff
@@ -122,6 +124,50 @@ def test_train_v2_import(monkeypatch, env_v2_enabled):
         assert Result is not ResultV2
 
 
+def test_exceptions_are_picklable():
+    """Test that TrainingFailedError and ControllerError are picklable."""
```

Comment:
Can you use `pytest.mark.parametrize("error", [TrainingFailedError(...), ControllerError(...)])` instead?